vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -119,16 +119,11 @@ capture_benchmark "${SCRIPT_NAME}_grant_access_to_disk"
 echo "Uploading $disk_resource_id to ${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd"
 
 echo "Setting azcopy environment variables with pool identity: $AZURE_MSI_RESOURCE_STRING"
-# TBD: Need to investigate why `azcopy-preview login --login-type=MSI` does not work for Windows
 export AZCOPY_AUTO_LOGIN_TYPE="MSI"
 export AZCOPY_MSI_RESOURCE_STRING="$AZURE_MSI_RESOURCE_STRING"
 
-if [[ "${OS_TYPE}" == "Linux" ]]; then
-  export AZCOPY_CONCURRENCY_VALUE="AUTO"
-  azcopy copy "${sas}" "${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd" --recursive=true || exit $?
-else
-  azcopy-preview copy "${sas}" "${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd" --recursive=true || exit $?
-fi
+export AZCOPY_CONCURRENCY_VALUE="AUTO"
+azcopy copy "${sas}" "${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd" --recursive=true || exit $?
```

Comment:
Should we comment that this is provided by the ADO pipeline step?  Which changed?

Reply:
Probably. I'm about to do test runs of all our pipelines today as I'm not confident they work after the ADO changes.